imap_processing/idex/constants.py

```diff
@@ -0,0 +1,10 @@
+"""Contains dataclasses to support IDEX processing."""
+
+TARGET_HIGH_CALIBRATION = 0.00135597
+ION_GRID_CALIBRATION = 0.00026148
+time_of_impact_init = 20
+constant_offset_init = 0.0
+rise_time_init = 3.71
+discharge_time_init = 37.1
+FILLVAL = -1.0e31
+TOF_Low_Peak_Prominence = 7
```

Comment:
```suggestion
TIME_OF_IMPACT_INIT = 20
CONSTANT_OFFSET_INIT = 0.0
RISE_TIME_INIT = 3.71
DISCHARGE_TIME_INIT = 37.1
FILLVAL = -1.0e31
TOF_LOW_PEAK_PROMINENCE = 7
```

Suggest to change this in order to be consistently using capitalized variables for constants (which is also suggested by PEP8)